### ISSUE TITLE
In Debt bug

### ISSUE THREAD

**Issue by AGulyaev12** [chummer5a/chummer5a#4367]
So here's the case

0) Careere mode
1) Have In Debt (Negative Quality) Rank 2 (or more)
2) Have enough karma to delete it
3) Try to byuout the quality
4) Only 2 karma are spent as if the system thinks that you've got only 1 Rank in this quality

As the Quality also requires money to byuout it would be nice to implement a feature that in order to delete this quality you also need a sum of money.

**Comment by DelnarErsike** [comment]
So, did you lower the level by 1? Or did you press the Remove Quality button, but it only removed one rank?

Also, what version of Chummer is this for?

**Comment by AGulyaev12** [comment]
It's 214 version. I pressed the "Remove quality" button. It completely removed the quality from the table.

I had 10 Ranks in this quality. I accumulated 20 Karma to remove it. I pressed the "Remove quality" button. It removed only 2 Karma from a character, as if the Quality had had only 1 Rank. Prior to remove the Quality the characted had only 16 Karma and the message said that there wasn't enough karma to remove the quality.
